test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
```

Comment:
> You are not deploying "ruby-ex s2i image", you are deploying the ruby-ex application. s2i is the process of converting source to image. That's only a small part of what's going on.

I had a misconception on s2i. Thanks for the correction. For readability propose we can add comment line in the feature file as 
```
# A user can deploy the example Ruby application ruby-ex
    User can login to OpenShift
    User can create new project
    User can deploy the example Ruby application ruby-ex
    User can delete project
# End of Ruby application ruby-ex deployment
```

> 

That said, I think we can spit this ling scenario up into several smaller ones:

    User can login to OpenShift
    User can create new project
    User can deploy the example Ruby application ruby-ex
    User can delete project

Having multiple scenarios will also help with the "When", "Then", "And" logic

Multiple scenario also helps to avoid resource problem for more application deployment. For example considering the single big scenario if i am deploying 3 application sequentially and 1st application failed to deploy, may be there is a chance of failure due to less resource available to the 2nd app and so on. Because the failed app will not be removed as it skips the step. By spiting the the bigger scenario we can overcome to this problem